src/ray/rpc/object_manager/object_manager_client.h

```diff
@@ -56,13 +68,15 @@ class ObjectManagerClient {
                    const ClientCallback<FreeObjectsReply> &callback) {
     client_call_manager_
         .CreateCall<ObjectManagerService, FreeObjectsRequest, FreeObjectsReply>(
-            *stub_, &ObjectManagerService::Stub::PrepareAsyncFreeObjects, request,
-            callback);
+            *stubs_[rand() % num_connections_],
+            &ObjectManagerService::Stub::PrepareAsyncFreeObjects, request, callback);
```

Comment:
Could we use a counter instead of rand()? Rand seems like it could be potentially slow.

Reply:
Let me switch to round-robin.